src/infrastructure/database/models/review/review_database_handler.py

```diff
@@ -0,0 +1,234 @@
+from bson import ObjectId
+from datetime import datetime
+from pymongo.collection import Collection, ReturnDocument
+
+
+from src.domain.user import UserBase
+from src.domain.review import ReviewCreate
+from src.domain.alcohol import AlcoholBase
+from src.domain.review.review_update import ReviewUpdate
+from src.infrastructure.database.models.review import Review
+
+
+class ReviewDatabaseHandler:
+    @staticmethod
+    async def get_alcohol_reviews(
+            collection: Collection[Review],
+            limit: int,
+            offset: int,
+            alcohol_id: str
+    ) -> list[Review]:
+        print(list(collection.find({'alcohol_id': ObjectId(alcohol_id)}).skip(offset).limit(limit)))
+        return (
+            list(collection.find({'alcohol_id': ObjectId(alcohol_id)}).skip(offset).limit(limit))
+        )
+
+    @staticmethod
+    async def count_alcohol_reviews(
+            collection: Collection[Review],
+            alcohol_id: str
+    ) -> int:
+        return (
+            collection.count_documents(filter={'alcohol_id': {'$eq': ObjectId(alcohol_id)}})
+        )
+
+    @staticmethod
+    async def check_if_alcohol_exists(
+            collection: Collection[AlcoholBase],
```

Comment:
To się nie zgadza.
AlcoholBase to byt domenowy.
W bazie nie musi to mieć przełożenia.
Więc primo powinno to być tylko `: Collection`, a ultimo to powinno znajdować się w handlerze alkoholu
Te klasy  `DatabaseHandler` to nic innego jak repozytoria (w javce się tego używa w ciul).
Tak więc jedno repozytorium = działania na jednej kolekcji/tabeli